### ISSUE TITLE
decompose() freaks when nested

### ISSUE THREAD

**Issue by danni** [danni/django-postgres-composite-types#12]
Model decl:

``` python
    activity = ArrayField(Activity.Field(), default=list)
```

Generated migration:

``` python
        migrations.AlterField(
            model_name='badger',
            name='activity',
            field=fields.ArrayField(base_field=postgres_composite_types.ActivityField(), default=list, size=None),
        ),
```

Which is where we declare this type :(

**Comment by danni** [comment]
Fixed by #14.

**Comment by danni** [comment]
In v0.3.3
